assets/js/queue_common.js

```diff
@@ -0,0 +1,52 @@
+var displayDate = new Date(),
+	monthStrings = [ 'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec' ];
+
```

Comment:
We could consider using a library like Moment for stuff like this, looks fine though.